astropy/io/fits/fitsrec.py

```diff
@@ -1284,6 +1284,17 @@ def _scale_back_ascii(self, col_idx, input_field, output_field):
         if 'D' in format:
             output_field[:] = output_field.replace(b'E', b'D')
 
+    def tolist(self):
+        # In some cases, at least when some columns are VLF, np.recarray.tolist (i.e. of the base class)
+        # can not interpret what is stored in data as list, and instead silently returns undersirable result.
+        # For example in the case of VLF column it an array with variable row length and offsets.
```

Comment:
Could you rephrase to something simpler, e.g. "Override .tolist to take care of VLF" and remove the TODO comment below ?

Reply:
Without explaining what's special about VLF? Ok.